creator-node/src/routes/files.js

```diff
@@ -837,7 +837,18 @@ router.post(
  * @dev This route does not handle responses by design, so we can pipe the response to client.
  * TODO: It seems like handleResponse does work with piped responses, as seen from the track/stream endpoint.
  */
-router.get(['/ipfs/:CID', '/content/:CID'], getCID)
+router.get(
+  ['/ipfs/:CID', '/content/:CID'],
+  async (req, _res, next) => {
+    if (!req.is('image/png') && !req.is('image/jpeg')) {
```

Comment:
I think this should be image/* || application/json

but tbh i think this should sit for a couple months before we merge. this data will all go thru discovery node eventually & storage v2